tock/utilization/views.py

```diff
@@ -67,11 +68,18 @@ class UtilizationAnalyticsView(PermissionMixin, TemplateView):
     def get_context_data(self, **kwargs):
         context = super(UtilizationAnalyticsView, self).get_context_data(**kwargs)
 
-        # use a date before Tock as the default start_date
-        start_date = self.request.GET.get("start", "2014-01-01")
-        end_date = self.request.GET.get("end", date.today().isoformat())
+        # use one year ago as the default start date
+        d = date.today()
+        start_date = self.request.GET.get("start", d.replace(year=d.year - 1).isoformat())
+        end_date = self.request.GET.get("end", d.isoformat())
         context.update({"start_date": start_date, "end_date": end_date})
 
+        # give a tip about the oldest possible date
+        ReportingPeriod = apps.get_model("hours", "ReportingPeriod")
```

Comment:
We only need to retrieve model classes from `apps` if we're working with them in a migration and we've already imported `ReportingPeriod` here so we're good to go! 
```suggestion
```